diagnostic.md

```diff
@@ -26,7 +27,10 @@ Record your responses inside the fenced code blocks below each question.
     the syntax for loading this component inside that template?
 
     ```html
-    <!-- your response here -->
+    {{#each contacts.contact as |contact|}}
+      {{contact/my-contact contact=contact}}
+    {{/each}}
+    //{{path-to-componet var-to-be-passed=var-in-current-file}}
```

Comment:
In this situation we want to access `model` instead of `contacts.contact`.  you don't need `contact/` in the sub-component.
```
{{#each model as |contact|}}
{{my-contact contact=contact}}
{{/each}}
```